src/main/java/ru/javarush/quest/controller/LogicServlet.java

```diff
@@ -0,0 +1,78 @@
+package ru.javarush.quest.controller;
+
+import ru.javarush.quest.entity.Answer;
+import ru.javarush.quest.entity.Question;
+import ru.javarush.quest.service.Context;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet(name = "LogicServlet", value = "/logic/*")
+public class LogicServlet extends HttpServlet {
+    private static final String ACCOUNT_NAME = "account_name";
+    private static final String QUESTION = "question";
+    private static final String COR_ANSWER = "cor_answer";
+    private static final String WRO_ANSWER = "wro_answer";
+
+    private static int count = 0;
+    private final Context context = Context.getDaoControllerInstance();
+
+    private static synchronized void getCount() {
+        count++;
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        if (count > 3) count = 0;
```

Comment:
`3` is magic number

Reply:
problem solved. for number 3 i added context.getQuestionSize() method. It is game level size.